### PR TITLE
Feat/otc support1

### PR DESCRIPTION
**Pull request by garloff**
This PR includes:
* environment files for otc (working) and otc-physical (WIP)
* README.otc
* renaming of neutron.patch into neutron-az.patch (patch required on half of the clouds)
* new neutron-snat.patch (patch required for OTC)
* new docker-ansible-otc.patch (patch req. for OTC Ubuntu images)

All in the terraform directory of testbed.

### CODE REVIEW

File: terraform/README.otc [hunk 1]
```diff
@@ -0,0 +1,25 @@
+Hints how to get SCS/OSISM testbed working on OTC
+=================================================
+
+- Use ENVIRONMENT=otc, referencing the included environment-otc.tfvars.
+  dhcp needs to be enabled and the DNS nameserver is set to the OTC
+  DNS (with quad9 fallback).
+- As on many other clouds, you need to apply neutron-az.patch to disable
+  network AZ hints
```
Comment (berendt) [inline terraform/README.otc:8]: Solved with overrides (#595).  No longer required.

File: terraform/README.otc [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+- You need to add "enable_snat = true" to the router in neutron.tf, you
+  can do so by applying the patch neutron-snat.patch (after the above-
+  mentioned patch).
```
Comment (berendt) [inline terraform/README.otc:11]: Solved with overrides (#595).  No longer required.

File: terraform/neutron-az.patch [hunk 3]
```diff
@@ -1,3 +1,14 @@
+From: Kurt Garloff <scs@garloff.de>
+Subject: Disable all availability zone hints for networking resources
```
Comment (berendt) [inline terraform/neutron-az.patch:2]: Solved with overrides (#595).  No longer required.

File: terraform/neutron-snat.patch [hunk 4]
```diff
@@ -0,0 +1,23 @@
+From: Kurt Garloff <scs@garloff.de>
+Subject: Enable SNAT on the router (for OTC)
```
Comment (berendt) [inline terraform/neutron-snat.patch:2]: Solved with overrides (#595).  No longer required.

File: terraform/terraform-0.13.patch [hunk 5]
```diff
@@ -0,0 +1,32 @@
+From: Kurt Garloff <scs@garloff.de>
+Subject: Adjust terraform to work with version 0.13
+
+We don't use any new features from 0.14 yet, so we can easily change
+the syntax to work with 0.13. The way providers are specified has changed,
+thus this is only a patch and not integrated into the main branch.
```
Comment (berendt) [inline terraform/terraform-0.13.patch:6]: Why? Makes no sense to me to enable per patch 0.13. We should always use the latest version of Terraform. This is also tested in the CI.

File: terraform/manager.tf [hunk 6]
```diff
@@ -236,6 +236,7 @@ write_files:
     path: /root/manager.sh
     permissions: 0700
 runcmd:
+  - "dpkg -r ansible || true"
```
Comment (berendt) [inline terraform/manager.tf:239]: Solved with #596. No longer required.

File: terraform/nodes.tf [hunk 7]
```diff
@@ -159,6 +159,7 @@ write_files:
     path: /root/node.sh
     permissions: '0700'
 runcmd:
+  - "dpkg -r ansible || true"
```
Comment (berendt) [inline terraform/nodes.tf:162]: Solved with #596. No longer required.

**Review by berendt — CHANGES_REQUESTED** [review]

File: terraform/environment-otc.tfvars [hunk 8]
```diff
@@ -0,0 +1,11 @@
+availability_zone         = "eu-de-02"
```
Comment (berendt) [inline terraform/environment-otc.tfvars:1]: Please move to environments/... (see #597).

**Comment by garloff** [comment]
Superceded by #601.

**Pull request closed without merging**